### ISSUE TITLE
Explain what happens with empty path in NUGET_PACKAGES variable

### ISSUE THREAD

**Issue by cmeeren** [MicrosoftDocs/azure-devops-docs#12037]
The guidelines contains this:

```yaml
NUGET_PACKAGES: ''
```

A previous version of the guidelines contained some other path.

Why the empty path? What does that mean?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 94a47e28-d827-dd09-61f4-7198941d2cb5
* Version Independent ID: 94a47e28-d827-dd09-61f4-7198941d2cb5
* Content: [Caching NuGet packages - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/artifacts/caching-nuget?view=azure-devops)
* Content Source: [docs/pipelines/artifacts/caching-nuget.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/main/docs/pipelines/artifacts/caching-nuget.md)
* Product: **devops**
* Technology: **devops-cicd-artifacts**
* GitHub Login: @ramiMSFT
* Microsoft Alias: **rabououn**

**Comment by WilliamAntonRohm** [comment]
@cmeeren -- Christer, thank you for your feedbacks.  Please consider these resources:
- [Azure DevOps Services](https://azure.microsoft.com/support/devops/) 
- [Azure DevOps on Stack Overflow](https://stackoverflow.com/questions/tagged/azure-devops)

Related to #12034, #12035, and #12036.

**Comment by cmeeren** [comment]
I realize that this is a question, but I think that the meaning of "special" values like empty strings should be explained in the official docs where it's used. So I propose adding a note to the doc page about this.

**Comment by cmeeren** [comment]
According to the official docs on [Pipeline caching#Path input](https://docs.microsoft.com/en-us/azure/devops/pipelines/release/caching?view=azure-devops#path-input), relative paths in `path` in the cache task are resolved against the working directory. I therefore assume that an empty string would mean that the entire working directory is cached, including all build output.

This of course just seems plain wrong. Particularly in a guide about caching NuGet packages.

I therefore suggest that the docs be fixed by setting the `NUGET_PACKAGES` variable (which is what is used in the `path` input) be set to the path of the actual downloaded NuGet packages. I'm fairly sure a previous version of the guide had it set to `NUGET_PACKAGES: $(Pipeline.Workspace)/.nuget/packages`. I propose using that instead of an empty string.

In other words; this issue is no longer a question, but a request for a change to the docs (just like the issues linked to above, though they are all otherwise unrelated changes).

**Comment by cmeeren** [comment]
By the way, if you agree with my request for change, I can create simple PRs to fix this and the other linked issues. I just wanted to make sure that I was on the right track before creating a PR, and therefore created an issue first.

**Comment by cmeeren** [comment]
Note also that [Pipeline caching#.NET/NuGet](https://docs.microsoft.com/en-us/azure/devops/pipelines/release/caching?view=azure-devops#netnuget) uses `$(Pipeline.Workspace)/.nuget/packages` as the path. So that supports my point that this should also be changed in the page related to this issue.

(There are other differences, too, which should perhaps be made consistent; namely, the presence of `restoreKeys`.)

**Comment by ramiMSFT** [comment]
@cmeeren Thanks for your feedback! I'm updating this shortly
